components/filter/filter.js

```diff
@@ -505,6 +517,56 @@ class Filter extends FocusMixin(LocalizeCoreElement(RtlMixin(LitElement))) {
 			listLabel = dimension.headerText;
 		}
 
+		let selectedListItems = nothing;
+		let listItems = nothing;
+		if (dimension.selectedFirst) {
+			if (listLabel) listLabel = this.localize('components.filter.selectedFirstListLabel', { headerText: dimension.headerText });
+			selectedListItems = dimension.values.filter(item => item.selectedOnRender).map(item => html`
+				<d2l-list-item
+					?selection-disabled="${item.disabled}"
+					?hidden="${item.hidden}"
+					key="${item.key}"
+					label="${item.text}"
+					selectable
+					?selected="${item.selected}">
+					<div class="d2l-filter-dimension-set-value d2l-body-compact">
+						<div class="d2l-filter-dimension-set-value-text">${item.text}</div>
+						${item.count !== undefined ? html`<div class="d2l-body-small">(${formatNumber(item.count)})</div>` : nothing}
+					</div>
+				</d2l-list-item>	
```

Comment:
Looks like all three of these are the same and only needs access to the `item`, so would be good to have a helper function like `_createSetDimensionItem(item)` or something.